packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -168,6 +174,10 @@ export const toggleTor = (isEnabled: boolean) => async (dispatch: Dispatch, getS
         if (!res) return;
     }
 
+    const progressStatus = isEnabled ? TorStatus.Enabling : TorStatus.Disabling;
```

Comment:
Yes, I was also thinking about it, `is` does not fit here at all, 'should' is much better because it about incoming status, not an existing one 👍 

b36d0dc72